packages/lib/server/repository/eventType.ts

```diff
@@ -185,4 +185,27 @@ export class EventTypeRepository {
       });
     }
   }
+
+  static async findAllByUserId({ userId }: { userId: number }) {
+    return await prisma.eventType.findMany({
+      where: {
+        userId,
+      },
+    });
+  }
+
+  static async findAllByTeamIdIncludeManagedEventTypes({ teamId }: { teamId?: number }) {
+    return await prisma.eventType.findMany({
+      where: {
+        OR: [
+          {
+            teamId,
+          },
+          {
+            parentId: teamId,
```

Comment:
We are mistakenly querying for eventType using `parentId=teamId.` parentId is supposed to be parent event's ID which has its teamId that we should compare with.